rules/sensio/tests/Rector/ClassMethod/TemplateAnnotationToThisRenderRector/Fixture/compact_controller.php.inc

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Rector\Sensio\Tests\Rector\ClassMethod\TemplateAnnotationToThisRenderRector\Fixture;
+namespace Rector\Sensio\Tests\Rector\FrameworkExtraBundle\TemplateAnnotationToThisRenderRector\Fixture;
```

Comment:
@TomasVotruba I'm not sure why when ClassMethod namespace is used in this fixture the test is failing. I have no clue why :/

Reply:
What's inside the rule that depends on this namespace?

Reply:
What's inside the rule that depends on this namespace?